lib/rules/selector-combinator-space-before/__tests__/index.js

```diff
@@ -143,6 +143,18 @@ testRule(rule, {
     {
       code: "namespace|type#id > .foo {}, space|customtype#id_withunder > a {}",
       description: "qualified id with namespace selector list"
+    },
```

Comment:
Can you move this tests into their own `testRule` with `syntax: scss` as they are without a nesting selector?